gradio/ipython_ext.py

```diff
@@ -0,0 +1,75 @@
+try:
+    from IPython.core.magic import register_cell_magic
+except ImportError:
+    pass
+
+import gradio
+
+
+def load_ipython_extension(ipython):
+    demo = gradio.Blocks()
+
+    @register_cell_magic
+    def blocks(line, cell):
+        if "imp" in line:
```

Comment:
what is this checking?

Reply:
If you do %%blocks imp then it becomes even easier to do development as all of the top-level gradio classes become imported: 

<img width="894" alt="image" src="https://user-images.githubusercontent.com/1778297/169162864-8ef03156-2c89-4cca-bcfe-92eef0995c92.png">